resources/views/clients/index.blade.php

```diff
@@ -0,0 +1,117 @@
+                                        {{__('Nr. of Projects')}}
+                                    </th>
+                                    <th scope="col" class="relative px-6 py-3 text-right">
+                                        @if(Auth::user()->hasRole('admin'))
+                                            <a href="/clients/create"
+                                               class="bg-gray-600 border border-transparent rounded-md shadow-sm py-2 px-4 inline-flex justify-center text-sm font-medium text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500">
+                                                {{__('Add new')}}
+                                            </a>
+                                        @else
+                                            {{__('Not entitled to create')}}
+                                        @endif
+                                    </th>
+                                </tr>
+                                </thead>
+                                @if(count($clients)>0)
+                                    <tbody>
+                                    <!-- Odd row -->
+                                    @foreach($clients as $client)
+                                        <tr class="@if($loop->index%2==0)bg-white @else bg-gray-50 @endif">
+                                            <td class="px-6 py-4 whitespace-nowrap text-sm font-medium text-gray-900">
+                                                {{$client->name}}
+                                            </td>
+                                            <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                                                {{$client->vat_id}}
+                                            </td>
+                                            <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                                                {{$client->full_address}}
+                                            </td>
+                                            <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                                                {{$client->projects_count}}
+                                            </td>
+                                            <td class="px-6 py-4 whitespace-nowrap text-right text-sm font-medium">
+                                                @if(Auth::user()->hasRole('admin') ||
+                                                    (Auth::user()->hasRole('user') && (int) Auth::id() === (int) $client->created_by))
+
+                                                    <a href="{{url('/clients/'.$client->id.'/edit')}}"
+                                                       class="text-indigo-600 hover:text-indigo-900 inline-flex">{{__('Edit')}}</a> &nbsp;
+                                                    @if($client->projects->count()==0)
```

Comment:
@faxunil I think this will cause N+1 query. Haven't tested, but it should be probably `$clients->projects()->count()`
My video: https://www.youtube.com/watch?v=MbksBczShcA